### DISCUSSION TITLE
Automatic height adaption in LineChart when client height changes?

### DISCUSSION THREAD

**Discussion by AlexKlueh** [Megabit/Blazorise#2399] [Q&A]
Currently, the LineChart remains the aspect ratio, which keeps the height the same when the parent container height changes:

![grafik](https://user-images.githubusercontent.com/9930148/119131315-945da780-ba39-11eb-9732-4aadeab14ddc.png)

After height increase:
![grafik](https://user-images.githubusercontent.com/9930148/119131510-dd156080-ba39-11eb-9a14-e9de6a95c505.png)

Is it possible to change the settings so that the charts also increases in height when the parent container does?

Best regards,

Alexander

**Comment by stsrki** [comment]
I think this will be possible with the fixes done lately for the aspect ratio option. https://github.com/Megabit/Blazorise/pull/2325

It will be available in v0.9.4, or preview if you want to try it.
